src/Data/MessagePack/RPC.hs

```diff
@@ -0,0 +1,94 @@
+-- | Implementation of [MessagePack RPC](https://github.com/msgpack-rpc/msgpack-rpc/blob/master/spec.md)
+
+module Data.MessagePack.RPC (
```

Comment:
How about renaming into `Data.MessagePack.RPC.Types`, and move `ClientViaWebSockets` into the RPC directory?
To tell the truth, I think the name `Network.MessagePack.ClientViaWebSockets` was a failure

Reply:
I would like to create `Network.MessagePack.Async.Client` to contain `callRpc`, `Client`, etc.

Reply:
Do you mean you are trying to put `callRpc` at `Async` directory though `callRpc` actually runs synchronously?
I think functions is `ClientViaWebSockets` are "asynchronous" only internally.

Reply:
Yes.
See the commits below.

Reply:
In the protocol level, it's async.